packages/api/src/schema/schema.yaml

```diff
@@ -964,6 +979,8 @@ components:
           readOnly: true
           example: https://livepeercdn.com/asset/ea03f37e-f861-4cdd-b495-0e60b6d753ad/index.m3u8
           description: URL for HLS playback
+        playbackPolicy:
+          $ref: "#/components/schemas/playback-policy"
```

Comment:
`$` 🧐

Reply:
wait what am I missing :eyes: isn't the $ right there?

Reply:
Yeah it is, twas just a historical joke lol